prometheus-operator/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+NAMESPACE=monitoring
+
+# Create the namespace for prometheus-operator
+kubectl create namespace ${NAMESPACE}
+
+# Create CRDs due to a bug. More information on https://github.com/helm/charts/tree/master/stable/prometheus-operator#helm-fails-to-create-crds
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/alertmanager.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheus.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/podmonitor.crd.yaml
+
+# Update your local Helm chart repository cache
+helm repo update
+
+# Install the prometheus-operator Helm chart
+helm install \
+  --name prometheus-operator \
+  --namespace ${NAMESPACE} \
+  --set prometheusOperator.createCustomResource=false \
+  stable/prometheus-operator
```

Comment:
Likewise is there a way of using a particular version? I know it means maintenance work when it's time to upgrade, but at least it means we have a tested/known-working version.